pan3d/ui/coordinate_configure.py

```diff
@@ -55,62 +60,95 @@ def __init__(
-                                    input=(
-                                        coordinate_change_slice_function,
-                                        f"[{coordinate_info}.name, 'start', $event.target.value]",
-                                    ),
-                                    __events=[("input", "input.prevent")],
-                                )
-                                vuetify.VTextField(
-                                    model_value=("%s?.stop" % coordinate_info, 0),
-                                    label="Stop",
-                                    hide_details=True,
-                                    density="compact",
-                                    type="number",
-                                    min=("%s?.start" % coordinate_info, 0),
-                                    max=("%s?.range[1]" % coordinate_info, 0),
-                                    __properties=[("min", "min"), ("max", "max")],
-                                    input=(
-                                        coordinate_change_slice_function,
-                                        f"[{coordinate_info}.name, 'stop', $event.target.value]",
-                                    ),
-                                    __events=[("input", "input.prevent")],
-                                )
-                                vuetify.VTextField(
-                                    model_value=("%s?.step" % coordinate_info, 0),
-                                    label="Step",
-                                    hide_details=True,
-                                    density="compact",
-                                    type="number",
-                                    min="1",
-                                    max=("%s?.size" % coordinate_info, 0),
-                                    __properties=[("min", "min"), ("max", "max")],
-                                    input=(
-                                        coordinate_change_slice_function,
-                                        f"[{coordinate_info}.name, 'step', $event.target.value]",
-                                    ),
-                                    __events=[("input", "input.prevent")],
-                                )
+
+                        vuetify.VCardSubtitle("Select values", classes="mt-3")
+                        with vuetify.VContainer(
+                            classes="d-flex pa-0", style="column-gap: 3px"
+                        ):
+                            vuetify.VTextField(
+                                model_value=("%s?.start" % coordinate_info, 0),
+                                label="Start",
+                                hide_details=True,
+                                density="compact",
+                                type="number",
+                                min=(
+                                    "parseFloat(+ %s?.range[0].toFixed(2))"
+                                    % coordinate_info,
```

Comment:
f-string

Reply:
Changed in [28561cb](https://github.com/Kitware/pan3d/pull/33/commits/28561cb331878bed9f5131df5a01d8307360c41e)